### PR TITLE
Phyplus release 1.2

### PR DESCRIPTION
**Pull request by chrade**

### CODE REVIEW

**Comment by xiaoxiang781216** [comment]
Please fix the warning not related to ROM Mix case symbol:
```
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:49:62: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:50:60: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:51:61: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:52:63: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:53:66: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:54:61: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:55:62: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:56:65: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:57:65: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:58:66: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:59:66: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:60:65: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:61:64: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:62:64: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:63:64: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:64:66: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:65:65: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:66:65: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:69:63: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:70:66: warning: Wrong column position of comment right of code
Warning: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/include/phy62xx/phy62xx_irq.h:71:65: warning: Wrong column position of comment right of code
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/src/phy62xx/phyplus_timerisr.h:1:77: error: Carriage returns found.  First detected
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/src/phy62xx/phyplus_timerisr.h:1:78: error: Long line found
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/src/phy62xx/phyplus_timerisr.h:2:1: error: Relative file path does not match actual file
Error: /home/runner/work/incubator-nuttx/incubator-nuttx/nuttx/arch/arm/src/phy62xx/pplus_mtd_flash.c:380:0: error: No indentation line
```

File: arch/arm/include/phy62xx/phy62xx_irq.h [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+ * under the License.
+ *
+ ****************************************************************************/
+
+/* This file should never be included directly but, rather, only indirectly
+ * through nuttx/irq.h
+ */
+
+#ifndef __ARCH_ARM_INCLUDE_STM32F0L0G0_STM32F0_IRQ_H
+#define __ARCH_ARM_INCLUDE_STM32F0L0G0_STM32F0_IRQ_H
```
Comment (LingaoM) [inline arch/arm/include/phy62xx/phy62xx_irq.h:26]: What this?

File: arch/arm/src/phy62xx/Kconfig [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	prompt "only PHY 6222 Chip Selection"
+	default ARCH_CHIP_PHY6222
+	depends on ARCH_CHIP_PHY62XX
+
+config ARCH_CHIP_PHY6222
+	bool "PHY6222"
+endchoise
+
+config PHY6222_BLE
+	bool "enable ble"
```
Comment (LingaoM) [inline arch/arm/src/phy62xx/Kconfig:18]:
```suggestion
	bool "Enable ble"
```

Comment (LingaoM) [reply arch/arm/src/phy62xx/Kconfig:18]: Below also look like too.

**Review by xiaoxiang781216 — COMMENTED** [review]
LGTM..

**Review by xiaoxiang781216 — APPROVED** [review]

**Pull request merged**